### ISSUE TITLE
Turn modal dialogs about certificates modeless 

### ISSUE THREAD

**Issue by justfortherec** [saenzramiro/rambox#589]
For some reason Rambox very frequently displays Certification Warnings (although invalid authority certificates are enabled; but that is another issue). This problem seems to be a recurring one and is really annoying because it does not only interrupt my reading in Rambox, but also clutters my display with huge modal dialogs.

Hence, my request for an enhancement: **Please show certificate warnings in modeless windows instead of modal ones** (e.g. in a red bar that pops up at the bottom of the Rambox window).

**Comment by rokups** [comment]
Proper fix is to make sure security warnings do not happen. Silencing them is a wrong fix.

**Comment by justfortherec** [comment]
I do not understand: What do you mean with "silencing"? Modeless dialogs do not silence the warnings but display them in a less intrusive way: They would not stop you from interacting with the entire Rambox window but still be displayed (say in a message bar that pops up at the bottom or top of the window).

**Comment by rokups** [comment]
Alright maybe that was not a best word to use. Look at it as a car whose engine will not turn on when critical defect is detected. You needing to go somewhere is not a good enough reason to risk blowing up. Currently rambox works as intended - invalid certificate means insecure communications and that is prevented. You want to work around a security feature. Not a good idea. Not at all... What we actually need is a new update with updated electron. But noone seems to care. So here we are.

**Comment by justfortherec** [comment]
@rokups, I totally agree that some security warnings should be blocking. I believe that is what you mean. However this only makes sense if the user gets the option to do anything about the problem. This is not the case the way how Rambox displays the warning: One can dismiss the button -- nothing else (i.e. there is no way for interaction either to prevent Rambox to make the insecure connection or allow it; instead, there is only an OK button to acknowledge whatever Rambox does anyway).

Since there is no choice for the user anyway, we can also just inform her about the certificate warning in a non-blocking way (and with 'non-blocking' I mean not blocking user experience; I am not speaking about insecure connections). If you are eager for Rambox not to connect to untrusted hosts, you have to make sure that the default action is to not make the connection (regardless of how this is shown in user interface).

To summarize: I believe the warnings that Rambox is displaying should be made modeless. Of course, that does not mean that the root cause of the warnings should not be fixed in the first place.

**Comment by rokups** [comment]
Hmmz maybe i misunderstood, sorry if that is the case. As long as security warning persists and connection is blocked its all good. Somehow i thought you meant for a way to dismiss warning and connect.

**Comment by saenzramiro** [comment]
Thank you guys. I was thinking about a way to display the warning to alert the user but not to disturb the usability. Next release (in a couple of days) will have this. Please wait!
